securedrop/translations/messages.pot

```diff
@@ -0,0 +1,18 @@
+# Translations template for SecureDrop.
+# Copyright (C) 2017 Freedom of the Press Foundation
+# This file is distributed under the same license as the SecureDrop project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: SecureDrop 0.3.12\n"
```

Comment:
I spy a hardcoded version string! If we cannot make this a Jinja var, then it must be added to the `./update_version.sh` script so that it gets incremented during the normal release process.

Reply:
This is a generated file (from ./manage.py translate) so it is updated every time it is re-generated.

The only reason for committing it is to communicate with the weblate server. It goes like this:

* strings changes in code and templates
* they are [committed to messages.pot](https://docs.securedrop.org/en/latest/development/i18n.html#updating-strings-to-be-translated)
* weblate.securedrop.club pulls the repository and updates its indices with the content of messages.pot

Does this seem like an acceptable workflow to you ?